projects/prison-custody-status-to-delius/src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/PcstdIntegrationTransactionRollbackTest.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.springframework.test.context.TestPropertySource
+import uk.gov.justice.digital.hmpps.data.generator.BookingGenerator
+import uk.gov.justice.digital.hmpps.data.generator.InstitutionGenerator
+import uk.gov.justice.digital.hmpps.data.generator.NotificationGenerator
+import uk.gov.justice.digital.hmpps.data.generator.PersonGenerator
+
+@TestPropertySource(properties = ["logging.level.org.springframework.transaction=DEBUG", "logging.level.org.hibernate.engine.transaction.internal.TransactionImpl=DEBUG"])
+class PcstdIntegrationTransactionRollbackTest : PcstdIntegrationTestBase() {
+    private val releaseOnLicence = "Released on Licence"
```

Comment:
Is this used?

Reply:
I use it at the moment to check the rollback statement in the logs.  In future, I would like to expand this test to assert on the rollback statement, see below:

2024-06-11T10:34:41.613+01:00 DEBUG 27958 --- [   scheduling-1] o.h.e.t.internal.TransactionImpl         : On TransactionImpl creation, JpaCompliance#isJpaTransactionComplianceEnabled == false
2024-06-11T10:34:41.613+01:00 DEBUG 27958 --- [   scheduling-1] o.h.e.t.internal.TransactionImpl         : begin
2024-06-11T10:34:41.954+01:00 DEBUG 27958 --- [   scheduling-1] o.h.e.t.internal.TransactionImpl         : **rolling back**
2024-06-11T10:34:41.955+01:00 ERROR 27958 --- [   scheduling-1] o.s.s.s.TaskUtils$LoggingErrorHandler    : Unexpected error occurred in scheduled task

org.springframework.dao.IncorrectResultSizeDataAccessException: Query did not return a unique result: 2 results were returned

Reply:
Sorry I meant the `private val releaseOnLicence = "Released on Licence"` line.